megalinter/MegaLinter.py

```diff
@@ -244,7 +244,17 @@ def run(self):
             config.get(self.request_id, "PARALLEL", "true") == "true"
             and len(active_linters) > 1
         ):
+            for active_linter in active_linters:
+                pre_post_factory.run_linter_pre_commands(
+                    active_linter.master, active_linter, run_before_linters=True
+                )
+
             self.process_linters_parallel(active_linters, linters_do_fixes)
+
```

Comment:
Do you see a use case where we need it in post commands ?
if not you can remove this part

Reply:
I can't think of it right now but “it's free” to have the same functionality in both pre and post, isn't it? Better to have it than not to have it.

Reply:
What do you think @echoix?

Reply:
@bdovaz it's just that if we keep it, we have something flagged "runBeforeLinters" that runs... after the linters ^^
In that case i would suggest to add another command property "run_after_linters" so it remains meaningful in the user configuration :)

Reply:
Done!